src/PolykeyClient.ts

```diff
@@ -34,51 +50,66 @@ interface PolykeyClient extends CreateDestroyStartStop {}
 )
 class PolykeyClient {
   static async createPolykeyClient({
-    nodePath = config.defaultsUser.nodePath,
-    streamFactory,
-    streamKeepAliveTimeoutTime,
-    parserBufferByteLimit,
+    port,
+    host,
+    options = {},
     fs = require('fs'),
     logger = new Logger(this.name),
     fresh = false,
   }: {
-    nodePath?: string;
-    streamFactory: StreamFactory;
-    streamKeepAliveTimeoutTime?: number;
-    parserBufferByteLimit?: number;
+    port: number;
+    host: string;
+    options: DeepPartial<PolykeyClientOptions>;
     fs?: FileSystem;
     logger?: Logger;
     fresh?: boolean;
   }): Promise<PolykeyClient> {
     logger.info(`Creating ${this.name}`);
-    if (nodePath == null) {
+    const optionsDefaulted = utils.mergeObjects(options, {
+      nodePath: config.defaultsUser.nodePath,
+      tokenBase: config.paths.tokenBase,
+      callTimeoutTime: config.defaultsSystem.rpcCallTimeoutTime,
+      parserBufferSize: config.defaultsSystem.rpcParserBufferSize,
+    });
```

Comment:
Why is `tokenBase` an option? we don't configure this.

Also the defaults structure doesn't match the structure of `options` The resulting `optionsDefaulted` will look like
```
{
   nodePath?: string;
   tokenBase: string;
   rpc: {
     callTimeoutTime: number;
     parserBufferSize: number;
   };
   callTimeoutTime: number;
   parserBufferSize: number;
   wsConfig?: Partial<WebSocketConfig>;
}
```

Which is definitely not correct.

Reply:
Wdym by why is tokenBase an option? In the jests the tokenBase is used as config.paths.tokenBase, should it not be an option?